elements/pfe-tabs/src/pfe-tabs.scss

```diff
@@ -59,3 +60,31 @@ $pfe-tabs__panel--PaddingLeft-factor:  2;
   padding-right: #{pfe-var(container-padding)};
   padding-left: calc(#{pfe-var(container-padding)} * #{$pfe-tabs__panel--PaddingLeft-factor});
 }
+
+/// ===========================================================================
+/// TAB VARIANTS
+/// ===========================================================================
+
+:host([pfe-variant="primary"]) {
+  .tabs {
+    border-bottom: transparent;
+    border-right: transparent;
+    justify-content: space-between; 
+  }
+}
+:host([vertical][pfe-variant="primary"]) {
+  align-items: flex-start;
+} 
+
+:host([pfe-variant="secondary"]) {
+  .tabs {
+    border-bottom: transparent;
+    justify-content: space-between; 
```

Comment:
@chrisdo1 can this be `justify-content: flex-start` with some spacing between the tabs themselves? https://www.evernote.com/l/AA5D4bwiVMBHFpVEYDipP9id_yzCVdb95C4

Reply:
@starryeyez024 Thanks for reviewing! I have removed the `flex-start` values for primary and secondary horizontal tabs and set a small margin to the right of each tab.